modules/passkey/contracts/test/TestWebAuthnLib.sol

```diff
@@ -4,6 +4,12 @@ pragma solidity ^0.8.0;
 import {P256, WebAuthn} from "../libraries/WebAuthn.sol";
 
 contract TestWebAuthnLib {
+    function castSignature(bytes calldata signature) external pure returns (WebAuthn.Signature calldata data) {
+        bool success;
+        (success, data) = WebAuthn.castSignature(signature);
+        require(success, "WebAuthnLib: invalid signature");
```

Comment:
```suggestion
        require(success, "WebAuthnLib: invalid signature encoding");
```

Would that be more accurate?

Reply:
Yes, it would.

Reply:
Broke your PR by commiting the suggestion and not updating this in tests, fixed it